tests/python/frontend/mxnet/test_forward.py

```diff
@@ -1373,6 +1374,266 @@ def verify(data_shape, anchor_shape, stds=[1, 1, 1, 1], clip=-1, in_format="corn
     verify((1, 10, 4), (1, 10, 4), in_format="center")
 
 
+dtype_list = ['float64', 'float32', 'int64', 'int32', 'bool']
+
+def test_forward_npi_pad():
+    if not hasattr(mx.sym.np, 'pad'):
+        pytest.skip("mx.sym.np.pad hasn't been publish yet")
```

Comment:
do we need to update CI's mxnet version?

Reply:
I'm afraid we couldn't update CI's mxnet version now. The latest version of mxnet has made some changes that the test functions in tvm didn't updated with them. So the CI would fail if we update mxnet version now.
This is the error message when I try to run pytest for [tests/python/frontend/mxnet/test_forward.py](https://github.com/apache/incubator-tvm/pull/6054/files/bca8e36ebcbb37ffe6ebbfeccdb4f561bf2c7ab8#diff-c52350cf958bfb7689b1c7c2adfe1a13).
`E       AttributeError: module 'mxnet' has no attribute 'mod'`

Reply:
We will need to find a good way to support both mx 1.x and 2.0.

For marking the test, maybe use `@pytest.mark.skipif `?

Reply:
Why not use two separate pipelines to test with the different mxnet versions?

Reply:
@tqchen what do you think?

Reply:
skip ci for now sounds good